src/Components/test/testassets/Components.TestServer/RazorComponents/Pages/Error.razor

```diff
@@ -0,0 +1,36 @@
+﻿@page "/Error"
+@layout ErrorLayout
+@using System.Diagnostics
+
+<PageTitle>Error</PageTitle>
+
+<h1 class="text-danger">Error.</h1>
+<h2 class="text-danger">An error occurred while processing your request.</h2>
+
+@if (ShowRequestId)
+{
+    <p>
+        <strong>Request ID:</strong> <code>@RequestId</code>
+    </p>
+}
+
+<h3>Development Mode</h3>
+<p>
+    Swapping to <strong>Development</strong> environment will display more detailed information about the error that occurred.
+</p>
+<p>
+    <strong>The Development environment shouldn't be enabled for deployed applications.</strong>
+    It can result in displaying sensitive information from exceptions to end users.
+    For local debugging, enable the <strong>Development</strong> environment by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong>
+    and restarting the app.
+</p>
+
+@code {
+    [CascadingParameter] public HttpContext? HttpContext { get; set; }
```

Comment:
Is this formatting idiomatic for Blazor? I'd have expected the attribute to be on its own line.

Reply:
I think so, although or templates don't showcase it much.

Reply:
I've changed it in any case